3Drat/threejs/three.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
if you really don't want to use npm or a bundler, rather than committing your deps to the repo just call it from a cdn

add this to the head of your html, above your code's script tag.

```
<script src="https://unpkg.com/three@0.128.0/build/three.min.js"></script>